types/voting.ts

```diff
@@ -22,7 +22,7 @@ export type PriceRequestT = {
   identifier: string;
   ancillaryData: string;
   voteNumber: BigNumber | undefined;
-  correctVote?: number;
+  correctVote?: string;
```

Comment:
votes are now raw strings rather than numbers so we dont lose information when the number is really large